webapp/src/ts/modules/analytics/analytics-target-aggregates.component.ts

```diff
@@ -68,7 +72,12 @@ export class AnalyticsTargetAggregatesComponent implements OnInit, OnDestroy {
     this.subscriptions.add(selectorsSubscription);
   }
 
-  private getTargetAggregates() {
+  private async setFacilityId() {
+    const userFacilities = await this.userSettingsService.getUserFacility();
+    this.userFacilityId = userFacilities[0]?._id;
+  }
```

Comment:
The reason we are setting the `userFacilityId` is so that on first load, we always fetch Aggregate Targets of the `default` facility. When we had users with only one facility there was no need to set `userFacilityId ` and we could call `getAggregates` without an id.

Reply:
Right, but in the `getAggregates` code, there is logic to just use the default facility for the user if no facilityId is provided.  So, I guess we should either remove that logic from in there or remove this logic here, but it seems like we do not need both.... :thinking:

Reply:
You're right, the service handles that.